README.md

```diff
@@ -359,6 +373,7 @@ import TooltipOnElementComponent from 'ember-tooltips/components/tooltip-on-elem
 export default TooltipOnElementComponent.extend({
   effect: 'fade',
   side: 'bottom',
+  enableLazyRendering: true,
```

Comment:
Should this change still be made in `your-app/components/tooltip-on-element}}` as the code example shows?